tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -1206,13 +1220,13 @@ public function userHasSentHTTPMethodToUrlWithPassword(string $user, string $ver
 	 * @param string|null $user
 	 * @param string|null $verb
 	 * @param string|null $url
-	 * @param TableNode|null $body
+	 * @param TableNode|string|null $body
 	 * @param string|null $password
 	 *
-	 * @return void
+	 * @return ResponseInterface
 	 * @throws GuzzleException
 	 */
-	public function sendingToWithDirectUrl(?string $user, ?string $verb, ?string $url, ?TableNode $body, ?string $password = null): void {
+	public function sendingToWithDirectUrl(?string $user, ?string $verb, ?string $url, $body = null, ?string $password = null): ResponseInterface {
```

Comment:
I think we want all the parameters to have type.

Reply:
this is something we did a while ago https://github.com/owncloud/QA/issues/694, I don't know if `mixed` type is yet available in `php7.4` but let's not leave the `type of $body here` . can we maybe pre process the body and send the body in string format in this function maybe? But if the changes will be a lot we can do that in other PR's